web/src/ui/app/App.tsx

```diff
@@ -9,10 +9,13 @@ import { startSearching, stopSearching } from "../../state/app-slice";
 import FloorTray from "../tray/FloorTray";
 import { DragStateType } from "../../drag/DragStateTypes";
 import { addFloor, addPing, removeToken } from "../../state/board-slice";
-import { setActiveFloor } from "../../state/floor-tray-slice";
+import {
+  setActiveFloor,
+  removeIcon as removeFloorIcon
```

Comment:
It seems like it would be better to make the name more specific at the implementation, instead of here. Why is this done this way?

Reply:
that's fair, at the time it felt like because the method is exported from floor-tray-slice that was redundant, but since you brought it up I think it makes sense to change